### DISCUSSION TITLE
Pull Requests

### DISCUSSION THREAD

**Discussion by Sarkoxed** [cvc5/cvc5#9943] [Q&A] [answered]
Hi! I had a pull request last week, but it seems to me that no one has checked it. Are there any special requirements for prs or any problems with mine so that I can prepare a better one?

**Comment by ajreynol** [comment, marked as answer]
Sorry for the delay, we will look at it soon.

In the meantime, there are some requirements for contributions listed here: https://github.com/cvc5/cvc5/blob/main/CONTRIBUTING.md.
